src/routes/Welcome.tsx

```diff
@@ -151,22 +169,36 @@ export function Welcome(props: IProps) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [importStatus]);
 
-  const handleGoOnline = () => {
-    handleOfflineChange('online');
+  const handleGoOnlineCloud = () => {
+    handleOfflineChange('online-cloud');
+  };
+
+  const handleGoOnlineLocal = () => {
+    handleOfflineChange('online-local');
   };
 
   const handleGoOffline = () => {
     handleOfflineChange('offline');
   };
+
   const handleQuickOnline = () => {
-    localStorage.setItem('autoaddProject', 'true');
-    handleGoOnline();
+    if (!hasOfflineProjects) localStorage.setItem('autoaddProject', 'true');
```

Comment:
why are we checking for offline projects when the user is going online?

Reply:
I guess in both these cases we may just want to check if they have any projects in their local cache. I'll make these changes.

Reply:
but shouldn't it check online projects then for go online?